robot/control/Inc/MicroPackets.hpp

```diff
@@ -10,128 +10,167 @@
 // The "consumer" of the data must check if it's valid and
 // the data isn't too old (if applicable)
 
-// Robot body velocities and dribbler commands
-// Radio -> Motion Control
+/** @struct MotionCommand
+ * Contains robot body velocities and dribbler commands
+ *
+ * Passed from @ref RadioModule to @ref MotionControlModule
+ */
 struct MotionCommand {
-    bool isValid = false;
-    uint32_t lastUpdate;
+    bool isValid = false; /**< Stores whether given data is valid  */
+    uint32_t lastUpdate;  /**< Time that MotionCommand was sent (milliseconds) */
 
-    float bodyXVel; // m/s
-    float bodyYVel; // m/s
-    float bodyWVel; // rad/s
+    float bodyXVel;       /**< x-component of robot's linear velocity (m/s) */
+    float bodyYVel;       /**< y-component of robot's linear velocity (m/s) */
+    float bodyWVel;       /**< Angular velocity of robot about z axis (rad/s) */
```

Comment:
Maybe say for reference "Angular velocity (w-component)"

Reply:
The W was supposed to represent omega here when I originally wrote it. About the Z axis as commented makes sense or the the r component of the pqr vector for angular velocity makes sense. I'm not sure I agree with the w component though

Reply:
I agree with @JNeiger 's assessment here. W (omega, really) is a variable representing angular velocity. Truly, it should be a vector with specific components (theta, phi, or whatever greek symbols you most prefer)